openedx/features/idea/views.py

```diff
@@ -0,0 +1,21 @@
+# -*- coding: utf-8 -*-
```

Comment:
Is necessary to keep this comment? if not let's please remove it from all files.

Reply:
it is good to have to inform the developer about the encoding as we a `unicode_literals `